### ISSUE TITLE
[BUG] ships powered by solar panels do not work

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#5296]
23w29a
a ship that is entirely powered by solar panels will have a positive power balance, however, this energy will be unusable in flight 
![image](https://github.com/tsunamayo/Starship-EVO/assets/56371294/d17a6cb9-a7b3-4d30-9c0e-ae2149890b96)
(using peach's tie again)
![image](https://github.com/tsunamayo/Starship-EVO/assets/56371294/3d4f0f52-4ab5-4f3d-b8b7-3e4814b9f845)
 power bar remains empty and ship cannot be flown, if you add some reactors the ship will be able to regain the ability to move however it will quickly run out of energy and become uncontrollable again

**Comment by Regnion** [comment]
Solar panels are also very underpowered. A 7.5 meter solar panel only produces 212 units of power. It would be nice if they were doubled in energy value. As well, moving over 100km closer to the star from the ring only increase power output by like 1.7 energy units per panel. 532 square meters of solar panels only produces 15,052 units of energy. That's a huge area on any build, and isn't enough to power the shields necessary to protect a station that might use a huge 532 sq m solar panel array.

**Comment by ProPeach** [comment]
Yeah I just tried a super simple fresh ship, wondered if there was a problem specifically for older ships converted to Solar like the TIE Tiki posted, but this super simple ship does not move when you pilot it despite having plenty of power.
![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/c6696357-9103-4403-b4d9-3e8d96bfa0a3)

**Comment by tsunamayo** [comment]
@ProPeach @TIKIRobo Hi, issue is fixed for 23w45a. Thanks

@Regnion Not sure what do you mean not centered? Maybe you are referring to the fact that the border is smaller than the center section when stretching? If yes it is a limitation of how the tech work. Also yes it looks like a waffle 🧇 🧇 🧇
